### DISCUSSION TITLE
masih bingung cara implemen apinya di postman

### DISCUSSION THREAD

**Discussion by windugi** [kirimwa-id/forum#76] [Tanya Jawab] [answered]
halo min untuk di header nya tokennya penempatannya gimana selalu missing api token

**Comment by rioastamal** [comment, marked as answer]
- Pengetahuan dasar tentang HTTP API diperlukan, seperti bagaimana mengirim HTTP header dan sebagainya
- Cek https://developer.kirimwa.id/api/v1/ untuk dokumentasi API lengkap
- Cek #68 untuk contoh

**Reply by windugi** [reply]
oke sudah bang sudah bisa

**Comment by kafaa-dev** [comment]
Pakai teknologi/bahasa pemrograman apa?

**Reply by windugi** [reply]
php bang laravel

**Reply by kafaa-dev** [reply]
oh sudah bisa ya?

**Comment by kafaa-dev** [comment]
Kalau Postman bisa di **Tab Headers** :
![Tab Headers](https://user-images.githubusercontent.com/73120669/145707386-34becb10-99c7-4718-baf9-961ec15f771b.png)
Ganti `<my_token>` dengan Token API KirimWa.id .

atau bisa juga di **Tab Authorization** :
![Tab Authorization](https://user-images.githubusercontent.com/73120669/145707421-d0d2f051-d3ee-4cee-a923-dc413ae294fe.png)
Ganti `{{accessToken}}` dengan Token API KirimWa.id .
